tests/test_bin_builder.py

```diff
@@ -0,0 +1,178 @@
+from unittest import TestCase
+from unittest.mock import Mock, patch
+from roulette import Outcome, Wheel, BinBuilder
+
+
+class TestBinBuilder(TestCase):
+    def setUp(self):
+        self.bin_builder = BinBuilder()
+        self.wheel = Wheel()
+
+    def test_bins_are_filled_for_straight_bets(self):
+        straight_bet_odds = 35
+        straight_outcomes = {
+            0: Outcome("0", straight_bet_odds),
+            1: Outcome("1", straight_bet_odds),
+            36: Outcome("36", straight_bet_odds),
+            37: Outcome("00", straight_bet_odds)
+            }
+
+        self.bin_builder.build_bins_for_straight_bets(self.wheel)
+
+        for bin_index in straight_outcomes:
+            self.assertIn(straight_outcomes[bin_index], self.wheel.bins[bin_index])
+
+    def test_bins_are_filled_for_horizontal_split_bets(self):
+        split_bet_odds = 17
+        split_outcome_name = "1-2"
+        bin_index = 1
```

Comment:
if bin 2 is also expected to hold the outcome, better test that too

Reply:
Done.